src/NUnitFramework/framework/Constraints/CollectionItemsEqualConstraint.cs

```diff
@@ -72,6 +72,20 @@ public CollectionItemsEqualConstraint Using(IComparer comparer)
             return this;
         }
 
+#if (NET_4_0 || NET_4_5) && !PORTABLE
```

Comment:
It's also supported for compact framework 3.5, so it's only **not** supported by .NET 2.0.